WordPress/twentytwentyfive
# Enhancement: Make footers translatable.

**Description**

I am adding i18n functions for footers.

I'm still doubtful we should include `Twenty Twenty-Five` as a hardcoded string in some footers. I know this discussion happened before. I believe that now that we have the copyright block binding, that's a good candidate. Else, re-consider the site title.

**Screenshots**



N/A

**Testing Instructions**

1. Open the site editor
2. Try every footer
3. Check that the strings are correct.
